.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Prepare
+      run: |
+        sudo apt-get update
+        sudo apt-get install libefiboot-dev check grep ninja-build meson gcc-5 libblkid-dev lcov valgrind gnu-efi libefivar-dev python3-pip ruby ruby-dev rubygems python3-setuptools
```

Comment:
Doesn't look like we need the python bits these days perhaps?

Reply:
Yep, makes sense - I'll push a new version without it. thanks.

Reply:
Well setuptools and pip at least.